### PR TITLE
Change kafka implementation

### PR DESCRIPTION
**Pull request by semtexzv**
Changes kafka implementation to use new library. 

Do not merge, it should be beneficial to compare performance of evaluator before and after.

### CODE REVIEW

File: listener/listener_test.go [hunk 1]
```diff
@@ -13,10 +13,9 @@ func TestRunReaders(t *testing.T) {
 	nReaders := 0
 	wg := sync.WaitGroup{}
 	wg.Add(1)
-	runReaders(&wg, mqueue.CreateCountedMockReader(&nReaders))
+	runReaders(mqueue.CreateCountedMockReader(&nReaders))
```
Comment (Josca) [inline listener/listener_test.go:16]: Do we still need wg here?

**Review by Josca — COMMENTED** [review]
It looks good. But as you mentioned perf. testing... wouldn't it be useful to keep old implementation usable as well and allow switching between these two frameworks using env variable?

**Comment by semtexzv** [comment]
Main goal is to simplify the application, leave a lot of partition handling to the library. I don't want to complicate things further by using 2 libraries at the same time

**Comment by Josca** [comment]
@semtexzv ok, so just please process my "wg" note, I can approve it then.

**Review by Josca — APPROVED** [review]
Ok, I hope it will work. Thanks for big upgrade.

**Pull request merged**